proxy/README.md

```diff
@@ -0,0 +1,99 @@
+# `cc-proxy`
+
+`cc-proxy` is a daemon offering access to the VM agent to multiple clients on
+the host. It exposes an API accessible through an `AF_UNIX` socket.
+
+This socket can be found at: `${localstatesdir}/run/cc-oci-runtime/proxy.sock`.
+
+## Protocol
+
+The proxy protocol is composed of JSON messages: requests and responses. Each
+of these message is on a separate line. The snippet below is showing two
+requests (`hello` and `hyper` commands) and their corresponding responses:
+
+```
+{ "id": 0, "command": "hello", "data": { "containerId": "foo", "ctlSerial": "/tmp/sh.hyper.channel.0.sock", "ioSerial": "/tmp/sh.hyper.channel.1.sock"  } }
+{"id":0,"success":true}
+{ "id": 1, "command": "hyper", "data": { "name": "ping" }}
+{"id":1,"success":true}
+```
+
+Requests have 3 fields: `id`, `command` and `data`.
+
+```
+type Request struct {
+	Id      uint             `json:"id"`
+	Command string           `json:"command"`
```

Comment:
I'd call that one ID and not Command. Here you have commands, hyper commands, requests and responses. I find it a bit confusing.